assets/app/components/site/Pages/navigationJsonPageList.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PageListItem from './pageListItem';
+
+const navigationJsonContent = ({ site }) => {
```

Comment:
I'm happy following any agreed convention. I tend to try to structure code where it starts at the highest level and references functions declared below as I find it more legible. My internal monologue goes, "I get what this does, now I need to understand what the function it's calling does because its name is dreadfully confusing... <scrolls down a bit> Ah, here it is."